cmd/shield/main.go

```diff
@@ -1465,6 +1529,50 @@ func main() {
 	c.Alias("rm archive", "delete archive")
 
 	/**************************************************************************/
+	err := LoadConfig(*options.Config)
+	if err != nil {
+		ansi.Fprintf(os.Stderr, "\n@R{ERROR:} Could not parse %s: %s\n", *options.Config, err)
+		os.Exit(1)
+	}
+
+	// only check for backends + creds if we aren't manipulating backends/help
+	nonAPICommands := regexp.MustCompile(`(help|backends|list backends|ls be|create backend|c be|update backend|backend|use backend)`)
+	if !nonAPICommands.MatchString(strings.Join(command, " ")) {
+		DEBUG("Command: '%s'", strings.Join(command, " "))
+
+		if *options.Shield != "" || os.Getenv("SHIELD_API") != "" {
+			ansi.Fprintf(os.Stderr, "@Y{WARNING: -H, --host, and the SHIELD_API environment variable have been deprecated and will be removed in a later release.} Use `shield backend` instead\n")
+		}
+
+		if len(Cfg.Backends) == 0 {
+			backend := os.Getenv("SHIELD_API")
+			if *options.Shield != "" {
+				backend = *options.Shield
+			}
+			ansi.Fprintf(os.Stderr, "@C{Initializing `default` backend as `%s`}\n", backend)
+			err := Cfg.AddBackend(backend, "default")
```

Comment:
backend could be empty here, if the caller didn't supply `-h`, and there is no `$SHIELD_API` environment variable.

We should probably also test what happens when you `AddBackend("", "alias")`, in the Config tests.